magicschoolbus/Assets/_Code/Backend/MathUtilities.cs

```diff
@@ -0,0 +1,55 @@
+﻿using UnityEngine;
+
+namespace MSB
+{
+    public static class MathUtilities
+    {
+        /// <summary>
+        /// Rotates the input values from the camera's perspective to the player's perspective.
+        /// If the player presses left while the player's character is facing left, this function's output will be forward relative to the player character.
+        /// </summary>
+        /// <param name="input">The input to be remapped. x is horizontal. z is vertical.</param>
+        /// <param name="camera">The active camera's transform.</param>
+        /// <param name="player">The player character's transform.</param>
+        /// <returns>The remapped input such that x is the player character's left and right and y is the player character's forward and backward.</returns>
+        public static Vector3 TransformInputFromCameraToPlayer(Vector3 input, Transform camera, Transform player)
```

Comment:
This function will be really useful.  Looks like a really clean way to call it too.